core/cli/node/main.cpp

```diff
@@ -19,6 +20,26 @@ namespace fc::cli::cli_node {
           CMD("listen", Node_net_listen),
           CMD("peers", Node_net_peers),
       })},
+      {GROUP("client")({
+          CMD("retrieve", Node_client_retrieve),
+          CMD("import", Node_client_importData),
+          CMD("deal", Node_client_deal),
+          CMD("generate-car", Node_client_generateCar),
+          CMD("local", Node_client_local),
+          CMD("find", Node_client_find),
+          CMD("list-retrievals", Node_client_listRetrievals),
+          CMD("inspect-deal", Node_client_inspectDeal),
+          CMD("deal-stats", Node_client_dealStats),
+          CMD("list-deals", Node_client_listDeals),
+          CMD("balances", Node_client_balances),
+          CMD("get-deal", Node_client_getDeal),
+          CMD("grant-datacap", Node_client_grantDatacap),
+          CMD("check-client-datacap", Node_client_checkClientDataCap),
+          CMD("list-notaries", Node_client_listNotaries),
+          CMD("add-verifier", Node_client_addVerifier),
+          CMD("list-clients", Node_client_listClients),
+          CMD("check-notary-datacap", Node_client_checkNotaryDataCap),
```

Comment:
Should these be moved from `client` to `filplus`?

Reply:
Should, fixed.